<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>701</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PUM Interfejs komunikacyjny</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DeviceBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="DeviceBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="DeviceBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="DeviceBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="aConsole_2"/>
    <addaction name="aClose"/>
   </widget>
   <widget class="QMenu" name="menuPo_czenie">
    <property name="title">
     <string>Połączenie</string>
    </property>
    <addaction name="aConfigure"/>
   </widget>
   <widget class="QMenu" name="menuUrz_dzenia">
    <property name="title">
     <string>Urządzenia</string>
    </property>
    <addaction name="actionLista_urz_dze"/>
    <addaction name="actionUstaw_urz_dzenia"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuPo_czenie"/>
   <addaction name="menuUrz_dzenia"/>
  </widget>
  <action name="aConfigure">
   <property name="text">
    <string>Konfiguracja</string>
   </property>
  </action>
  <action name="actionKonsola">
   <property name="text">
    <string>Konsola</string>
   </property>
  </action>
  <action name="aConsole_2">
   <property name="text">
    <string>Konsola</string>
   </property>
  </action>
  <action name="actionZapisz_ustawienia">
   <property name="text">
    <string>Zapisz ustawienia</string>
   </property>
  </action>
  <action name="actionWczytaj_ustawienia">
   <property name="text">
    <string>Wczytaj ustawienia</string>
   </property>
  </action>
  <action name="actionWczytaj_urz_dzenie">
   <property name="text">
    <string>Wczytaj urządzenie</string>
   </property>
  </action>
  <action name="aClose">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionLista_urz_dze">
   <property name="text">
    <string>Lista urządzeń</string>
   </property>
  </action>
  <action name="actionWczytaj_urz_dzenie_2">
   <property name="text">
    <string>Wczytaj urządzenie</string>
   </property>
  </action>
  <action name="actionZapisz_urz_dzenie">
   <property name="text">
    <string>Zapisz urządzenie</string>
   </property>
  </action>
  <action name="actionPanel_kontrolny">
   <property name="text">
    <string>Panel kontrolny</string>
   </property>
  </action>
  <action name="aConsole">
   <property name="text">
    <string>Konsola</string>
   </property>
  </action>
  <action name="actionUstaw_urz_dzenia">
   <property name="text">
    <string>Ustaw urządzenia</string>
   </property>
  </action>
  <action name="actionDodaj_urz_dzenie">
   <property name="text">
    <string>Dodaj urządzenie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeviceBox</class>
   <extends>QGroupBox</extends>
   <header>devicebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
